scripts/convert-to-platforms.sh

```
@@ -1633,21 +1640,27 @@ convert_to_umbrel() {
         elif [[ "$port_spec" =~ ^[0-9]+$ ]]; then
             # Format: just the port number - use for both
             if [[ "$use_port_override" == false ]]; then
                 host_port="$port_spec"
             fi
-            container_port="$port_spec"
+            if [[ "$use_app_port_override" == false ]]; then
+                container_port="$port_spec"
+            fi
         else
             # Complex format (e.g., "8080:8000/tcp")
             local clean_spec=$(echo "$port_spec" | sed 's|/.*||')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)